test/inductor/test_torchinductor.py

```diff
@@ -3971,6 +3971,27 @@ def set_num_threads(num_threads):
         def test_timed_cpu_only(self):
             timed(lambda: torch.randn(10), ())
 
+        def test_complex_memory_overlap(self):
```

Comment:
What's the motivation of adding the complex_memory_overlap here? I thought the output layout is backend independent and no need to check specifically for CPU?

Reply:
Done.